docs/hpc/queues/job-sizing-guidance.md

```diff
@@ -84,7 +77,7 @@ If your job doesn’t need these flags then we recommend not adding them as they
 | ---- | ----- | ----------- |
 | writabletmp | True/False | Some applications need to write to /tmp, (cx3 only) |
 | filesystem_type | gpfs/nfs | Target only nodes with given filesystem type |
-| cpu_type | rome/skylake/haswell/ivy/sandy | Target only nodes with given CPU type |
+| cpu_type | rome | Target only nodes with given CPU type |
```

Comment:
If there is just one cpu_type, does this flag make sense? In that case, all jobs should be going to rome processors.

Reply:
Yes but this flag will become useful again once this is rewritten for Phase 2. A stronger argument could be made to remove Infiniband and NFS though.

Reply:
To be clear, you mean that it will help to select the icelake nodes once we rewrite it for CX3 phase 2. If yes, then you are right.

Reply:
yep that was my meaning